style.css

```diff
@@ -437,3 +449,103 @@ button.resume:visited {
   background: #36b37f;
   color: #fff;
 }
+
+.form_section {
+  padding: 0 24px;
+}
```

Comment:
- [ ] Please add the background image for this section to match your design with the Figma one
- Your Design
![image](https://user-images.githubusercontent.com/50941074/159816517-24e80865-e741-450b-9fa0-fb0750d8f4ab.png)

- The Figma Design
![image](https://user-images.githubusercontent.com/50941074/159816596-f9b488ca-9db4-4e9c-ad24-5e49eba8f080.png)